### PR TITLE
Several Bug Fixes and Issues Fixed

### PR DESCRIPTION
**Pull request by cstock123**
- Fixed app Crash on the Add To Calendar Button on Events Details
- Changed app Tint Color to CruDeepBlue
- Added filter to remove Events after the startDate has passed by 1 day
- Added sort method to Events to sort by Date

### CODE REVIEW

File: CruCentralCoast/Events/EventsVC.swift [hunk 1]
```diff
@@ -35,8 +35,17 @@ class EventsVC: UITableViewController {
...
+        let currentDate = Date()
+        let calendar = Calendar.current
+        //offset by 1 to keep the event for one more day
+        let dateOffset = calendar.date(byAdding: .day, value: -1, to: currentDate)
+        
         let subscribedMovements = LocalStorage.preferences.getObject(forKey: .subscribedMovements) as? [String] ?? []
-        self.dataArray = DatabaseManager.instance.getEvents().filter("ANY movements.id IN %@", subscribedMovements)
+        self.dataArray = DatabaseManager.instance.getEvents()
+            .filter("ANY movements.id IN %@", subscribedMovements)
+            .filter("endDate >= %@", dateOffset)
```
Comment (tydahlwave) [inline CruCentralCoast/Events/EventsVC.swift:46]: Nice use of multiple predicates 👏

**Comment by cstock123** [comment]
Here are some screenshots of the new appTintColor. LMK what you think.

screen shot 2018-09-29 at 11 28 14 pm
screen shot 2018-09-29 at 11 28 41 pm

**Comment by tydahlwave** [comment]
Only thing you may have missed for the tint color is the login/sign up/forgot password screens

**Review by tydahlwave — APPROVED** [review]

**Comment by tydahlwave** [comment]
Good to merge. A good habit is to reference which issues you closed in this PR or to reference this PR from the issues when you close them. If you use the phrase "Closed" or "Fixed" in front of "Pull Request" or "Issue" and followed by the # and number, GitHub will automatically link them

**Comment by cstock123** [comment]
Oh wow that is really nice and good to know! Will do that in the future for sure. Thank you

**Pull request merged**